src/components/LecturerCourse.vue

```diff
@@ -47,12 +55,21 @@
                 required: true,
             },
         },
-        setup(props: any) {
+        async setup(props: any) {
+            const auth: UserManagement = new UserManagement();
+
+            const responseHandler = new ProfileResponseHandler();
+            const response = await auth.getSpecificUser(props.course.lecturerId);
+            const user = responseHandler.handleReponse(response);
```

Comment:
Rename `user` to `lecturer` for maintainability. Whereas the question remains if the lecturer really needs to see his own name on his own course list?

Reply:
Not at the moment, but I think there will be a list of all courses visible for the lecturer in the future